compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1902,3 +1902,6 @@ error.cannot.infer.type.from.spread.op=\
 
 error.tuple.and.expression.size.does.not.match=\
   tuple and expression size does not match
+
+error.named.arg.not.allowed.with.included.record.param=\
+  named-arg ''{0}'' not allowed with included-record-param ''{1}''
```

Comment:
We shouldn't directly use productions from the grammar, these aren't things we expect the user to know.

Reply:
This error currently doesn't describe the actual issue anyway. It reads like we can't specify a named argument when there is an included record parameter in the function.

Reply:
Changed